README.md

```diff
@@ -45,7 +45,7 @@ Run the plugin with Acorn:
 
 ```shell
 # dev mode:
-acorn run --name acorn-istio-plugin -i .
+acorn run --name acorn-istio-plugin --profile dev -i .
```

Comment:
Shouldn't dev mode set the profile to dev?

Reply:
I'm not sure whether that happens automatically...but that's a good question

Reply:
Just tested it. You're right, it does 👍

Reply:
Sweet, I'll update the docs to be more explicit about this behavior.